pkg/skaffold/build/local/types.go

```diff
@@ -95,6 +95,10 @@ func (b *Builder) PushImages() bool {
 
 // Labels are labels specific to local builder.
 func (b *Builder) Labels() map[string]string {
+	return map[string]string{}
```

Comment:
```suggestion
	return nil
```
to make it consistent with `pkg/skaffold/build/gcb/types.go`

Reply:
actually we need to return an empty map here to avoid a nil pointer exception - so I need to change that other file, good catch :)